src/Stratis.Bitcoin.Features.Consensus/PowConsensusValidator.cs

```diff
@@ -162,6 +179,16 @@ public virtual void ContextualCheckBlock(ContextInformation context)
             this.logger.LogTrace("(-)[OK]");
         }
 
+        /// <summary>
+        /// Validates the UTXO set is correctly spent.
+        /// </summary>
+        /// <param name="context">The context.</param>
+        /// <param name="taskScheduler">The task scheduler.</param>
```

Comment:
task scheduler is task scheduler, but ?? 
such comments are not very helpful
i mean without going into the code, why would ExecuteBlock needs task scheduler? 

anyway, i'd recommend to refactor this method to get rid of the task scheduler here, because looking at the code, it seems like no one uses it anyway (or did i miss that?)